### DISCUSSION TITLE
Change permissions for OpenMage repos?

### DISCUSSION THREAD

**Discussion by sreichel** [OpenMage/magento-lts#2635] [General]
@Flyingmana can you please change the permissions for some repos like ModuleFostering or WebNotifaction?

For ModuleFostering I already had a working script to add modman/composer.json to every extension. Just got no permission to push this. Please change!

For WebNotifaction ... https://github.com/OpenMage/Web_Notifications/pull/4

**Comment by elidrissidev** [comment]
He's active in Discord more

**Comment by Flyingmana** [comment]
First of all,
I get notified for everything on Github, the downside is that mentioning me is not increasing visibility. I will just not be aware of it in most of the cases. If there is anything which needs my attention, then it needs someone sharing the link on the OpenMage Discord Server.


for the module Fostering, I dont know anymore where and when, but I already told you to submit the script to the tooling repository as a PR, I will not give out permissions there.

For the web notifications please ask someone with Admin permissions on the organization, or make a suggestion on how we should assign permission via an rfc.
With the low usage there, and the fact nobody felt the need to ping me via discord about it, I dont see the need to change anything there.

**Comment by sreichel** [comment]
> I will just not be aware of it in most of the cases.

This is only your problem Go to github -> Notifications (top menu) -> Mentioned/Assigned filter (left side), Is that hard to find?

> then it needs someone sharing the link on the OpenMage Discord Server.

From your post: https://www.openmage.org/2019/08/18/new-home-magento-connect-modules.html

_You want support us in maintaining, get in contact with us on **Github**, Twitter or via Discord._

 (btw https://www.openmage.org/blog-patches-and-addons ...)

> but I already told you to submit the script to the tooling repository as a PR

No. You told me to integrate it into "tooling".

No! Why? Why to change a working script that only run one time?

> the fact nobody felt the need to ping me via discord about it

The other way around. _"then it needs someone"_, _"nobody felt the need"_ ... Do you expect people to run after you?
